util/topgen.py

```diff
@@ -582,33 +583,19 @@ def generate_rstmgr(topcfg, out_path):
             outputs.append(rtl_path / Path(x))
 
     # Parameters needed for generation
-    clks = []
-    output_rsts = OrderedDict()
-    sw_rsts = OrderedDict()
-    leaf_rsts = OrderedDict()
+    reset_obj = topcfg['reset_obj']
```

Comment:
I'd consider adding a check that `isinstance(reset_obj, Resets)` here to give a quick explanation of what's going on if something comes unstuck.

Reply:
yep will add.

Reply:
I don't think the check is in the latest version?